code/game/objects/items/cards_ids.dm

```diff
@@ -512,9 +512,9 @@ update_label()
 		..()
 
 /obj/item/card/id/centcom
```

Comment:
You should add a new type for them instead.

Reply:
New ID cards are now made.